packages/ui/src/components/Session/PreSession/PreSession.tsx

```diff
@@ -60,19 +62,22 @@ export const PreSession: React.FC<Props> = ({
         "flex flex-col lg:grid lg:grid-cols-[1fr,auto] lg:grid-rows-[auto,1fr] items-center lg:items-start gap-4 lg:gap-6"
       )}
     >
-      <FocusedStream
-        muted
-        stream={{
-          stream,
-          speaking,
-          audio: audio.enabled,
-          video: video.enabled,
-          user: members.current,
-          cast: false,
-        }}
-      />
+      <div className="h-full w-full rounded-2xl flex items-stretch relative overflow-hidden min-h-[398px] bg-natural-100 md:min-h-[744px] md:h-[744px] lg:min-h-max lg:h-[550px]">
+        <Stream
+          muted
+          stream={{
+            stream,
+            speaking,
+            audio: audio.enabled,
+            video: video.enabled,
+            user: members.current,
+            cast: false,
+          }}
+        />
+      </div>
```

Comment:
The latest logic on master looks a little bit different: 

![image](https://github.com/user-attachments/assets/7b70010a-87ec-43d6-b698-f5bd7c3d33fb)